include/caf/power_manager.rst

```diff
@@ -0,0 +1,152 @@
+.. _caf_power_manager:
+
+Power manager module
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Use the power manager module to reduce the energy consumption of battery-powered devices.
+The module achieves this by switching to low power modes when the device is not used for a longer period of time.
+
+Configuration
+*************
+
+You can enable the power manager module by selecting the :option:`CONFIG_CAF_POWER_MANAGER` option in the configuration.
+
+This module uses Zephyr's :ref:`zephyr:power_management_api` subsystem.
+
+Timeout configuration options
+=============================
+
+With the :option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` configuration option, you can set the amount of time after which the application will go to the low power mode.
+By default, the timeout is set to 120 seconds.
+
+The :option:`CONFIG_CAF_POWER_MANAGER_ERROR_TIMEOUT` sets the amount of time after which the device is turned off upon an internal error.
+
+Optional boolean for keeping the system on
+==========================================
+
+The :option:`CONFIG_CAF_POWER_MANAGER_STAY_ON` lets the system stay on also when there are no active connections.
+
+For more information about configuration options, check the help in the configuration tool.
+
+Implementation details
+**********************
+
+The power manager is started when the "main" is ready (which is reported using ``module_state_event``).
+
+When started, it can do the following operations:
+
+* Manage `Power states`_
+* Trigger `Switching to low power`_
+* Handle `Wake-up scenarios`_ from suspended or off states
+
+Power states
+============
+
+The application can be in the following power states:
+
+* `Idle`_
+* `Suspended`_
+* `Off`_
+* `Error`_
+
+The power manager takes control of the overall operating system power state.
+See the following section for more details about how the application states converts to the system power state.
+
+Idle
+----
+
+In this state, the board is either connected or actively looking for a new connection (advertising).
```

Comment:
This is actually nRF Desktop specific too

Reply:
ok